index.html

```diff
@@ -22,14 +22,14 @@ <h1 id="title">
             <!-- Text section -->
             <div class="form-group">
                 <label for="name">Name</label>
-                <input type="text" name="name" id="name" class="formControl" placeholder="Enter your name" required>
+                <input type="text" name="name" id="name" class="formControl" placeholder="Enter your Full name" required>
             </div>
             <!-- end of text section -->
```

Comment:
Add a more descriptive pull request

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear eg why having a good PR description will benefit the student.